### DISCUSSION TITLE
docs/reference/crossdb-cli/

### DISCUSSION THREAD

**Discussion by giscus** [crossdb-org/docs#6] [Announcements]
# docs/reference/crossdb-cli/

The Fastest Embedded Database in the world

https://crossdb.org/docs/reference/crossdb-cli/

**Comment by sloanyyc** [comment]
怎么创建 索引呢？使用SQL,没有找到
How to create a Index on Table in SQL.

**Reply by jcwangxp** [reply]
create index idx_name on tbl_name (col_name,...)
